lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -2,7 +2,8 @@
 
 module SafePgMigrations
   module IdemPotentStatements
-    def add_index(table_name, column_name, **options)
+    def add_index(table_name, column_name, *args)
```

Comment:
I'm not sure, we have to change that. You can keep `**options`, and remove `ruby2_keywords` and it will works just fine, if I understood everything properly.

Even the Rails, the signature is like this: `def add_index(table_name, column_name, **options)` and they are forwarding like this `add_index_options(table_name, column_name, **options)`

Reply:
You're probably right.  It's possibly a little trickier I found because sometimes the other "Plugins" are passing around things in a way that actually wouldn't be passed to the real method.  I ended up changing everything earlier on because I wasn't sure.

Now, I'm split though.  To me, the question is: should we do blind delegation or not.  Take for example this scenario:
Rails adds a new optional positional argument to add_index.  Since this is not ordinarily a backwards incompatible change, they don't wait to drop it in Rails 7.  But we (and other users of our gem) are now incapable of taking advantage of that new option without updating this code.

Reply:
It's really good point. Let me some time to process that, and re-read the whole things with that in mind.